gcovr/tests/test_args.py

```diff
@@ -79,6 +79,41 @@ def test_invalid_objdir(capsys):
     assert c.exception.code == 1
 
 
+def helper_test_non_existing_directory(capsys, option):
+    c = capture(capsys, [option, 'not-existing-dir/file.txt'])
+    assert c.out == ''
+    assert 'The output directory does not exist: \'not-existing-dir\'' in c.err
+    assert c.exception.code != 0
+
+
+def test_non_existing_directory_output(capsys):
+    helper_test_non_existing_directory(capsys, '--output')
```

Comment:
`@pytest.mark.parametrize(…)` to the rescue? That way the helper could serve as the single test function.

Reply:
Ok.

Reply:
I've checked the `paramtrize` but didn't get it to run.

Reply:
ok, no problem